website/content/docs/upgrading/upgrade-specific.mdx

```diff
@@ -25,6 +25,12 @@ In previous versions, Consul emitted redundant state store usage metrics that co
 
 As of Consul v1.19, Consul does not emit the redundant metric with the double `consul.consul` in its name. Any monitoring alerts and dashboards that you may have utilizing these metrics may require edits to update to the simplified metric name.
 
+### Legacy API Gateway Removal
+In previous versions, Consul supported the helm chart stanza .Values.apiGateway. This was [deprecated in Consul v1.16.0](https://developer.hashicorp.com/consul/docs/release-notes/consul/v1_16_x#what-s-deprecated).
+As of Consul v1.19.0, this has been removed. If you are using the legacy API Gateway,
+you must migrate to [connectInject.apiGateway](https://github.com/hashicorp/consul-k8s/blob/main/charts/consul/values.yaml#L2368) before upgrading to Consul v1.19.0.
```

Comment:
Proposing a suggestion with the following goals:
- Move the required action earlier (in case someone is skimming)
- Make it clear that there's still a Consul API gateway, we're just removing the legacy version

Feel free to take/leave what you want:
```suggestion
The Consul Kubernetes Helm chart stanza
[`.Values.apiGateway`](/consul/docs/v1.16.x/k8s/helm#apigateway-1)
is removed in Consul v1.19.0. If using that stanza, migrate to the
[`connectInject.apiGateway`](/consul/docs/k8s/helm#v-connectinject-apigateway)
stanza before upgrading to Consul v1.19 or later.

The `.Values.apiGateway` stanza is associated with the legacy API gateway that was
[deprecated in Consul v1.16.0](https://developer.hashicorp.com/consul/docs/release-notes/consul/v1_16_x#what-s-deprecated).
The Kubernetes-only legacy API gateway is superseded by the modern, multi-runtime
[API gateway](/consul/docs/connect/config-entries/api-gateway).
On Kubernetes, the modern API gateway is associated with the `connectInject.apiGateway` stanza.
```